src/main/java/core/basesyntax/Application.java

```diff
@@ -3,5 +3,10 @@
 public class Application {
```

Comment:
The class 'Application' should have an access modifier. It's recommended to specify 'public' to make it clear that it's intentionally accessible from anywhere.